.vsts.ci.yml

```diff
@@ -166,6 +166,7 @@ jobs:
 - job: build_linux_arm64_agent
 ################################################################################
   displayName: Linux Agent (ARM64)
+  condition: false
```

Comment:
Do we want this to be where branch is not equal to master and doesn't start with release?

Reply:
That way we can get ci builds on a different branch for testing

Reply:
No, I don't want it running on CI builds for branches for now. We can remove these lines specifically in the branch where we continue testing Arm64.

Reply:
Ok, fair enough